lib/mock/mock-interceptor.js

```diff
@@ -106,7 +106,7 @@ class MockInterceptor {
     if (typeof data === 'undefined') {
       throw new InvalidArgumentError('data must be defined')
     }
```

Comment:
data can never be undefined because it is always set to empty string

https://github.com/nodejs/undici/blob/fedae35f1b71999d2327ef6a28538f5845a71515/lib/mock/mock-interceptor.js#L151